dashboard/test/controllers/api/v1/pd/workshop_survey_report_controller_test.rb

```diff
@@ -189,24 +189,6 @@ class WorkshopSurveyReportControllerTest < ::ActionController::TestCase
       params: -> {{workshop_id: @workshop.id}}
     )
 
-    test 'facilitators can see results for local summer workshops' do
```

Comment:
Delete because it doesn't really test if a facilitator can see results. It just tests if we routes the request to the correct processing function. That logic is now an 1-line check if the workshop course is either CSD or CSP.